.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+<!-- Please answer these questions before submitting your feature request. Thanks! -->
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Maybe it would be better to put this, and similar lines below, in comments, like in the other templates? To not require people to delete this line before they start filling their own contents

Reply:
Sure I can can make this change.

Reply:
I submitted #146 to fix this.